integration/service/oasis/stationgraph/edge.go

```diff
@@ -1,7 +1,16 @@
 package stationgraph
 
-type neighbor struct {
-	targetNodeID nodeID
-	distance     float64
-	duration     float64
+type edgeID struct {
+	fromNodeID nodeID
+	toNodeID   nodeID
+}
+
+type edge struct {
+	distance float64
+	duration float64
+}
+
+type edgeIDAndData struct {
```

Comment:
I'll prefer to name it `edge`. Name the the former `edge` to `metric` or something like this.

Reply:
Good suggestion.  Will consider to make the change in the next pull.